packages/backend-defaults/src/entrypoints/scheduler/database/tables.ts

```diff
@@ -20,8 +20,10 @@ export const DB_TASKS_TABLE = 'backstage_backend_tasks__tasks';
 export type DbTasksRow = {
   id: string;
   settings_json: string;
-  next_run_start_at: Date;
+  next_run_start_at?: Date | string; // This can be null when in manual trigger mode
```

Comment:
null as in `null`?

Reply:
In the database row NULL yes.

This is an interesting question overall actually; should the TS types for these rows also always have `| undefined | null` at the end when they are nullable? Because you MAY pass in undefined or null depending on circumstances when you do writes, and in reads, I guess they should come out as nulls or do they ever get coerced to undefined? Not sure. But yeah there may be a risk of accidentally doing `if (row.column !== undefined)` instead of `if (row.column)` if you don't add `null` as an explicit type option here.